perma_web/perma/settings/deployments/settings_common.py

```diff
@@ -157,7 +157,6 @@ def _pywb_static_dir_location():
 
```

Comment:
Do we still need/want this part?

  387: API_VERSION = 1
  388  
  389  TEMPLATE_VISIBLE_SETTINGS = (
  390:     'API_VERSION',